docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -41,4 +41,22 @@ As a result, only the trace for the `publish` and initial services are visible.
 
 In the trace details, you can see the tags for the `publish-service`.
 
-![](assets/troubleshoot-only-publish-detail.png)
\ No newline at end of file
+![](assets/troubleshoot-only-publish-detail.png)
+
+### Scenario: Configured microservice or lambda returns an error
+
+This scenario assumes that there is a microservice or lambda configured to recieve
+the event trigger. However, due to a bug in the code, the microservice or lambda 
+failed to process the event.
+
+As a result, one can see the `webhook`, `push` and `name-of-lambda` services in the trace and they are marked with error.
+
+![](assets/troubleshoot-error-in-lambda.png)
+
+One can see the error details by clicking on one of the service spans. e.g. span for `push` service
```

Comment:
``To see the error details, click on one of the service spans. For example, choose the span for the `push` service.``